### PR TITLE
Removing server side Currency, making number formatting clide-side an…

### LINKED ISSUE #45: Consolidate Lang and Money Format
Currently we have Language English, Number format in US and Currency in R$.

The funniest thing is that we only have pt-BR users.

Time to consolidate. Lets have everything in pt-BR then we add internationalization afterwards.

Marking as a bug just because I think it is bugged UI. No strong feelings here.

### PR DESCRIPTION
**Pull request by danielfireman**
…d using a library that consolidates based on the language. Also removed the collapsible elements, it was quite hard to get how it was working. Lets come up with a cleaner way. Fixes #45

### CODE REVIEW

File: static/app.template.html [hunk 1]
```diff
@@ -4,9 +4,6 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="icon" href="/static/favicon.ico" type="image/x-icon">
 
-		<!-- Bootstrap min is required for collapse -->
```
Comment (brunojdo) [inline static/app.template.html:7]: I think bootstrap is needed to the tooltip work properly.

File: static/app.template.html [hunk 2]
```diff
@@ -29,83 +26,95 @@
...
-    x1 = x1.replace(rgx, '$1' + ',' + '$2');
-  }
-  return x1 + x2 + ' ' + postSymbol;
-}
+		<script src="//cdnjs.cloudflare.com/ajax/libs/numeral.js/1.5.3/numeral.min.js"></script>
+
+<script type="text/javascript">
+// Loading language.
+// Extracted from github.com/adamwdraper/Numeral-js/blob/master/languages/pt-br.js
+numeral.language('pt-br', {
```
Comment (brunojdo) [inline static/app.template.html:36]: Nice

File: static/app.template.html (hunk 1 above)
Comment (brunojdo) [inline static/app.template.html:7]: as I suspected, this remove broke the tooltip UI. Look at http://isumm-canary.appspot.com/app

File: static/app.template.html [hunk 3]
```diff
@@ -29,83 +26,95 @@
...
+  },
+})
+
+// Setting language and default currency formatter.
+numeral.language('pt-br')
+numeral.defaultFormat('$ 0[.]0[,]00');
 
 function formatCurrency(n) {
-  return formatNumber(n, '{{.Currency}}', '')
+  return numeral(n.toFixed(2)).format()
```
Comment (brunojdo) [inline static/app.template.html]: Don't work in the isumm-canary deploy.

File: static/app.template.html [hunk 4]
```diff
@@ -29,83 +26,95 @@
...
 }
 
 function formatPercent(n) {
-  return formatNumber(n, '', '%')
+  return numeral(n).format('0,00%')
```
Comment (brunojdo) [inline static/app.template.html]: Now, the interval of the yaxis of the Interest Rate is between 0% and 100%. Before that, was between -1 and 1.

File: static/app.template.html (hunk 3 above)
Comment (danielfireman) [inline static/app.template.html]: It seems it does
currency

File: static/app.template.html (hunk 1 above)
Comment (danielfireman) [inline static/app.template.html:7]: Yep. Brought it back.

File: static/app.template.html (hunk 4 above)
Comment (danielfireman) [inline static/app.template.html]: Great catch. After some fiddling, I believe we are good to go.
localhost 8080 app

**Comment by danielfireman** [comment]
Thanks for the great review! Canary has been updated.

**Pull request merged**